src/metrics.ts

```
@@ -153,7 +153,8 @@ export const epochCoverageGauge = new Gauge({
 
 export const continuousObserverStateGauge = new Gauge({
   name: 'observer_continuous_state',
-  help: 'Current state of continuous observer (0=waiting, 1=observing, 2=finalizing)',
+  help:
+    'Current state of continuous observer (0=waiting, 1=observing, 2=finalizing, 3=expired)',
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)